app/helpers/adhoq/application_helper.rb

```diff
@@ -17,7 +17,7 @@ def schema_version
         ActiveRecord::SchemaMigration.maximum(:version)
       else
```

Comment:
the dependency is still `>= 3.2` 😉 
https://github.com/esminc/adhoq/blob/bd1a9cfbc62fa2f996ff16658b6f810de391a389/adhoq.gemspec#L19